ReactiveXComponent/Parser/XCApiConfigParser.cs

```diff
@@ -1,42 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Xml;
-using System.Xml.Linq;
 using ReactiveXComponent.Configuration;
 
 namespace ReactiveXComponent.Parser
 {
-    public class Parser
+    public class XCApiConfigParser : IXCApiConfigParser
```

Comment:
perhaps in your method names repository instead of dico or dictionary could be better. what do you think ?
open discussion